error[E0133]: use of extern static is unsafe and requires unsafe function or block

```
$ cargo check --features gecko-ffi
    Checking thin-vec v0.2.10 (/tmp/thin-vec)
error[E0133]: use of extern static is unsafe and requires unsafe function or block
    --> src/lib.rs:1536:47
     |
1536 |         self.ptr.as_ptr() as *const Header == &EMPTY_HEADER
     |                                               ^^^^^^^^^^^^^ use of extern static
     |
     = note: extern statics are not controlled by the Rust type system: invalid data, aliasing violations or data races will cause undefined behavior

For more information about this error, try `rustc --explain E0133`.
error: could not compile `thin-vec` due to previous error
```

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -1533,7 +1533,7 @@ impl<T> ThinVec<T> {
     #[cfg(feature = "gecko-ffi")]
     #[inline]
     fn is_singleton(&self) -> bool {
-        self.ptr.as_ptr() as *const Header == &EMPTY_HEADER
+        unsafe { self.ptr.as_ptr() as *const Header == &EMPTY_HEADER }
     }
 
     #[cfg(not(feature = "gecko-ffi"))]

